wifiphisher/pywifiphisher.py

```diff
@@ -583,6 +601,10 @@ def start(self):
                 print ("[{0}+{1}] Selecting {0}{2}{1} interface for the deauthentication "
                        "attack\n[{0}+{1}] Selecting {0}{3}{1} interface for creating the "
                        "rogue Access Point").format(G, W, mon_iface.get_name(), ap_iface.get_name())
+                # randomize the mac addresses
+                if not args.no_mac_randomization:
+                    ap_iface.randomize_interface_mac(args.mac_ap_interface)
+                    mon_iface.randomize_interface_mac(args.mac_deauth_interface)
```

Comment:
I just want to make the code cleaner when call this function; otherwise we need to distinguish whether user specified the mac address, so the code at pywifiphisher.py will become:
   ```
   if not args.no_mac_randomization:
       if not args.mac_ap_interface:
           ap_iface.randomize_interface_mac()
       else:
            ap_iface.randomize_interface_mac(args.mac_ap_interface)
   ```
And the same is for the deauth interface, how do you think?